strength_percentiles.py

```diff
@@ -30,55 +32,65 @@
 BENCH_COLUMN = 17 # ...
 DEADLIFT_COLUMN = 19 # ...
 
-# This dictionary will contain lists of tuples.  Each tuple will contain one
-# lifter's (squat, bench, deadlift, total). The lists divide the results
-# by the lifters' category (ex: men, amateurs) but each lifter has
-# their result repeated in four lists, ("all," one of "men" and "women," one
-# of "amateur" and "professional," and one of "raw" and "equipped")
-categorized_results = {'all': [],
-        'amateurs': [],
-        'professionals': [],
-        'men': [],
-        'women': [],
-        'raw': [],
-        'equipped': []}
-
-def scrape_webpage_and_store_meet_results(quote_page):
-    page = urllib2.urlopen(quote_page)
-    soup = BeautifulSoup(page, 'html.parser')
-    meet_results_table = soup.find_all('tr') # all rows of meet results table,
-                                             # each corresponds to an
-                                             # individual's performance at the
-                                             # meet
-    for row in meet_results_table: # information for one lifter
-        parse_and_store_result(row)
+def parse_result_and_categories(row):
+    """
+    Parses table row of BeautifulSoup object for categories (gender,
+    professional status, equipment) and result (squat, bench, deadlift) of
+    the associated lifter.
 
-def parse_and_store_result(row):
+    Parameters:
+    -----------
+    row: bs4.element.tag
+        A table row of BeautifulSoup object which contains category and result
+        information of a lifter in the expected format (see SQUAT_COLUMN,
+        BENCH_COLUMN, etc.)
+
+    Returns:
+    --------
+    result, categories: Result is tuple of ints, or None, containing the
```

Comment:
This return value of a 2-tuple consisting of a tuple in one slot and another tuple in the other slot is not ideal. A dictionary of the form:
{'squat': squat_num, 'bench': bench_num, .... }
would be a lot more logical and clean.